### PR TITLE
simplify ping_succeeds and firewall_disabled

### PR DESCRIPTION
**Pull request by bmwiedemann**
system call already returns a bool
so we can just return it as is

@rsalevsky might want to review this change of his code

### CODE REVIEW

File: crowbar_framework/lib/crowbar/checks/network.rb [hunk 1]
```diff
@@ -67,19 +67,11 @@ def ip_configured?
       end
 
       def firewall_disabled?
-        if system("sudo LANG=C iptables -n -L | grep -qvE '^$|^Chain [^ ]|^target     prot'")
-          return false
-        end
-
-        true
+        return ! system("sudo LANG=C iptables -n -L | grep -qvE '^$|^Chain [^ ]|^target     prot'")
```
Comment (aplanas) [inline crowbar_framework/lib/crowbar/checks/network.rb]: Maybe these spaces between "target" and "prot" and be expressed also as a regex (I know that is not related with this PR)

Comment (aplanas) [inline crowbar_framework/lib/crowbar/checks/network.rb]: Also I agree with houndci: the `return` is redundant

File: crowbar_framework/lib/crowbar/checks/network.rb [hunk 2]
```diff
@@ -67,19 +67,11 @@ def ip_configured?
...
       end
 
       def ping_succeeds?
-        unless system("ping -c 1 #{fqdn} > /dev/null 2>&1")
-          return false
-        end
-
-        true
+        return system("ping -c 1 #{fqdn} > /dev/null 2>&1")
```
Comment (aplanas) [inline crowbar_framework/lib/crowbar/checks/network.rb]: same

**Comment by rsalevsky** [comment]
The changes are not ruby style conform, from my point it also makes it not easier to understand.

**Comment by bmwiedemann** [comment]
addressed hound style issues

**Comment by dirkmueller** [comment]
@rsalevsky can you explain the ruby style issue?

**Comment by aplanas** [comment]
+1 for me

**Comment by vuntz** [comment]
+1, but would like to hear more about the style issue.

**Comment by rsalevsky** [comment]
It's fine now. +1

**Pull request merged**